src/main/java/org/snomed/snowstorm/core/data/services/identifier/IdentifierCacheManager.java

```diff
@@ -154,6 +158,24 @@ public void populateIdBlock(IdentifierReservedBlock idBlock, int quantityRequire
 				cache.unlock();
 				requestSatisfied = true;
 			}
+		} else {
+			//If no cache available & not requesting for International (as already prefetched),
+			//then prefetch additional identifiers for subsequent requests.
+			int prefetchQuantity;
+			switch (componentType) {
+				case Concept:
+					prefetchQuantity = conceptIdPrefetchCount;
```

Comment:
@kaicode , I have copied the prefetch quantity for Concepts, Descriptions and Relationships from Config::getIdentifierCacheManager. Are you happy with the numbers?

Reply:
yeah, the numbers are fine thanks